docs/destructuring.md

```diff
@@ -2,59 +2,65 @@
 title: Destructuring
 ---
 
-"Destructuring" is a visually concise way of extracting fields from a data structure. You can use destructuring anywhere you'd normally use a variable.
+Destructuring is a concise way to extract data from a structure. Destructuring
+can be used anywhere a variable name might appear.
 
-## Usage
-
-The following binds variables: `ten = 10`, `twenty = 20`
+## Tuples
 
 ```reason
-let someInts = (10, 20);
-let (ten, twenty) = someInts;
-```
+let pair = (1, "Hello");
 
-The following binds variables: `name = "Guy"`, `age = 30`
```

Comment:
I kind of prefer single line comments for inline code example comments. For some reason it looks a little cleaner/familiar.

Reply:
I do too, but the syntax highlighting isn't supported. I'm happy to go back and update them all if support is added

Reply:
Another consideration is that sketch doesn't support single line comments. If we want examples to be copy/pastable in there then we should stick to `/* */` style comments or get support added in sketch.